.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
unexpected?

Reply:
this is intentional to disable guttering which is duplicate of `Run tests| Update tests`